config/single_texturemapped.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": 1,
+  "masteraddress": "localhost",
+  "users": [
+    {
+      "eyeseparation": 0.065,
+      "pos": { "x": 0, "y": 0, "z": 0.0 }
+    }
+  ],
+  "settings": {
+    "display": { "swapinterval": 0 }
+  },
+  "nodes": [
+    {
+      "address": "localhost",
+      "port": 20401,
+      "name": "DSGP1",
+      "swaplock": false,
+      "windows": [
+        {
+	  "fullscreen": false,
```

Comment:
The indentation in this file is a bit mixed up

Reply:
There were some tabs hiding in there